scheds/rust/scx_rusty/src/bpf/intf.h

```diff
@@ -43,12 +44,10 @@ enum consts {
 	NSEC_PER_SEC            = NSEC_PER_USEC * USEC_PER_SEC,
 
 	/* Constants used for determining a task's deadline */
-	DL_RUNTIME_SCALE	= 2, /* roughly scales average runtime to */
-				     /* same order of magnitude as waker  */
-				     /* and blocked frequencies */
-	DL_MAX_LATENCY_NS	= (50 * NSEC_PER_MSEC),
+	DL_RUNTIME_FACTOR	= 100,
 	DL_FREQ_FT_MAX		= 100000,
 	DL_MAX_LAT_PRIO		= 39,
+	DL_FULL_DCYCLE		= 650000,
```

Comment:
How did you determine the optimal value fo `DL_FULL_DCYCLE`? I'm wondering if this should be a function of the time slice... like, what happens if I start rusty using a different slice_ns?

Reply:
It would be a good idea to document how to determine DL_FULL_DCYCLE (or idea behind the DL_FULL_DCYCLE), which, I guess, can be determined relatively to the maximum time slice.

Reply:
So, I actually determined it from looking at the output of a task with duty cycle approximately = 1. The rust implementation gives a proper float, but we're stuck with using fixed-point integer arithmetic in the kernel. I'll add a comment explaining this.

Reply:
So, it's not really related to time slice. It tracks the period of time during which a task is runnable, as a proxy for it running / its duty cycle in some time interval. We accumulate these in rusty_quiescent() and rusty_runnable(). I'll add a comment with a bit more explanation.